Client/Client.Qt/Application.hpp

```diff
@@ -19,7 +20,8 @@ enum class AppState
 {
     LoginForm,
     RegistrationForm,
-    ChatWindowForm
+    ChatWindowForm,
+    SettingsWidgetForm
```

Comment:
First things first it's not a form

Reply:
Could you please elaborate a bit more why it should be named differently, and how do you suggest to call it instead?

Reply:
Form is a thing that switches in main window